packages/theme-patternfly-org/components/tableOfContents/tableOfContents.js

```diff
@@ -10,25 +10,57 @@ export const TableOfContents = ({ items }) => {
     const { innerWidth } = window;
     innerWidth !== width && setWidth(innerWidth);
   }
+  let wasSublistRendered = false;
 
-  const renderItem = (item, index) => {
-    return Array.isArray(item)
-      ? (
-        <JumpLinksList key={index} className="ws-toc-sublist">
-          {item.map(renderItem)}
+  const renderSublist = (item, nextItem, idx) => {
+    wasSublistRendered = true;
+    return (
+      <>
+        {item.text}
+        <JumpLinksList>
+          {nextItem.map(curItem => (
+            <JumpLinksItem
+              key={curItem.id}
+              href={`#${curItem.id}`}
+              className="ws-toc-item"
+              onKeyDown={updateWidth}
+              onMouseDown={updateWidth}
+              onClick={() => trackEvent('jump_link_click', 'click_event', curItem.id.toUpperCase())}
+            >
+              {curItem.text}
+            </JumpLinksItem>
+          ))}
         </JumpLinksList>
-      ) : (
+      </>
+    );
+  }
+
+  let jumpLinksItems = [];
+
+  const renderJumpLinksItems = () => {
+    items.forEach((item, index) => {
+      let cur = item;
```

Comment:
Should `cur` be utilized anywhere within this function, or could this line be removed?

Reply:
Same as above - removed this unused var 👍